service/frontend/adminHandler.go

```diff
@@ -248,7 +248,7 @@ func (adh *adminHandlerImpl) DescribeWorkflowExecution(
 	}
 
 	shardID := common.WorkflowIDToHistoryShard(request.Execution.WorkflowID, adh.numberOfHistoryShards)
-	shardIDstr := string(shardID)
+	shardIDstr := strconv.Itoa(shardID)
```

Comment:
nit: we can drop one of shardID strings here

Reply:
I looked into this one a bit more (and failed to make a smaller repro), and have decided to undo it / change it to a form that `go vet` allows (and is recommended in the github issue thread that lead to the vet check being added) but produces the same output as before.

Since this appears to change how shards in a hash ring are selected, is it safe to change at all?  Or could it cause Problems™ on deploy, as different instances will compute different shards for the same keys?